mods/stairs/init.lua

```diff
@@ -46,12 +46,24 @@ end
 -- Node will be called stairs:stair_<subname>
 
 function stairs.register_stair(subname, recipeitem, groups, images, description, sounds)
+	local stair_images = {}
+	for i, image in ipairs(images) do
+		if type(image) == "string" then
+			stair_images[i] = {
+				name = image,
+				backface_culling = true,
+			}
+		elseif image.backface_culling == nil then
+			stair_images[i] = table.copy(image)
```

Comment:
Is this correct? or
`stair_images = table.copy(images)`?
or
`stair_images[i] = images[i]`?